### PR TITLE
issue-36: Create branch 11.0.0

### PR DESCRIPTION
**Pull request by jperedadnr**
- Source/Target compatibility bumped to 1.9
- Core can be used as dependency on any project with Java 9, 10, 11
- Sampler runs standalone in JDK: 9, 10, 11
- Added Launcher class to Sampler to allow running via shadow jar
- Remove private API

### CODE REVIEW

**Comment by Birdasaur** [comment]
Excellent work as usual Jose.

File: FXyz-Samples/build.gradle [hunk 1]
```diff
@@ -34,36 +38,50 @@ mainClassName = fxyz_sampler_mainClass
...
-sourceSets {
-    main {
-        compileClasspath += configurations.jdk
-    }
-}
-
 dependencies {
+    compile "org.openjfx:javafx-graphics:11:win"
+    compile "org.openjfx:javafx-graphics:11:linux"
+    compile "org.openjfx:javafx-graphics:11:mac"
```
Comment (abhinayagarwal) [inline FXyz-Samples/build.gradle]: Is adding all 3 platform a requirement for the fat jar?

File: build.gradle [hunk 2]
```diff
@@ -25,14 +35,17 @@ subprojects {
...
 
-    sourceCompatibility = '1.8'
-    targetCompatibility = '1.8'
-
     repositories {
         jcenter()
     }
     
     dependencies {     
+        compile "org.openjfx:javafx-base:11:${platform}"
```
Comment (abhinayagarwal) [inline build.gradle]: You can get rid of the curly braces surrounding platform

File: FXyz-Samples/build.gradle (hunk 1 above)
Comment (jperedadnr) [reply FXyz-Samples/build.gradle]: Yes, this adds the native libs of all three platforms to the fat jar.
If you don't do this, when you create the jar you will have only the current platform, so the jar won't be cross-platform

Comment (abhinayagarwal) [reply FXyz-Samples/build.gradle]: Thanks for clarifying.

**Pull request merged**